tests/secrets_and_cfgmap_test.go

```diff
@@ -0,0 +1,229 @@
+package tests_test
+
+import (
+	"flag"
+	"time"
+
+	expect "github.com/google/goexpect"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/pborman/uuid"
+
+	"kubevirt.io/kubevirt-ansible/tests"
+	"kubevirt.io/kubevirt/pkg/config"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	"kubevirt.io/kubevirt/pkg/log"
+	ktests "kubevirt.io/kubevirt/tests"
+)
+
+var _ = Describe("Config", func() {
+
+	flag.Parse()
+
+	BeforeEach(func() {
+		ktests.BeforeTestCleanup()
+	})
+
+	Context("With a Secret and a ConfigMap defined", func() {
+
+		Context("With a single volume", func() {
+			var (
+				configMapName string
+				configMapPath string
+				secretName    string
+				secretPath    string
+			)
+
+			virtClient, err := kubecli.GetKubevirtClient()
+			ktests.PanicOnError(err)
```

Comment:
initialization if _virtClient_ can be move to first context (line 28) and reuse in both cases

Reply:
Fixed this. https://github.com/kubevirt/kubevirt-ansible/pull/478/files#diff-2f400e67bdfdcf16ebaa3fbafd688302R21